completion_aggregator/tasks/aggregation_tasks.py

```diff
@@ -243,13 +243,15 @@ def update_for_aggregator(self, block, affected_aggregators, force):
                     aggregation_name=block.block_type,
                     earned=total_earned,
                     possible=total_possible,
+                    percent=total_earned / total_possible,
```

Comment:
I guess there isn't any possibility for total_possible to be zero here, since it wouldn't reach this code in that case?

Reply:
I added a chapter with no children, which demonstrated that this does indeed cause problems, so I fixed the calculation.

Reply:
Thanks for catching that.